packages/editor/src/plugins/rows/components/add-row-button-floating.tsx

```diff
@@ -1,56 +1,72 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
 import { faCirclePlus } from '@fortawesome/free-solid-svg-icons'
-import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
-import { useEditorStrings } from '@serlo/frontend/src/contexts/logged-in-data-context'
 import { cn } from '@serlo/frontend/src/helper/cn'
 import { useRef } from 'react'
 
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+
 interface AddRowButtonFloatingProps {
-  onClick: React.MouseEventHandler<HTMLButtonElement>
+  onClick: (event: React.MouseEvent | React.KeyboardEvent) => void
   focused?: boolean
   hide?: boolean
 }
 
 export function AddRowButtonFloating({
-  focused,
   onClick,
+  focused,
   hide,
 }: AddRowButtonFloatingProps) {
   const rowsStrings = useEditorStrings().plugins.rows
   const buttonRef = useRef<HTMLButtonElement>(null)
 
+  if (hide) return null
+
+  const baseStyles = `
+  group z-50
```

Comment:
What is the `group` and `relative` for?

Reply:
I think the group stuff was remnants from the first implementation I did.

It appears that if I remove relative, the new z-index change on hover `hover:z-50` doesn't work

Reply:
A nice article that explains the reason behind it: https://www.joshwcomeau.com/css/stacking-contexts/